next/styles/globals.css

```
@@ -1,9 +1,34 @@
+  --color-text-muted: #777;
+  --color-accent: #3291ff;
+  --color-border: #333;
+  --color-code-bg: #1e1e1e;
+  --color-pre-bg: #1a1a1a;
+}
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
```

Stylelint (17.6.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)